src/test/TestProperty.cpp

```diff
@@ -0,0 +1,213 @@
+
```

Comment:
Blaspheme! No Koppywright?!

// Copyright © 2008-2021 Pioneer Developers. See AUTHORS.txt for details
// Licensed under the terms of the GPL v3. See licenses/GPL-3.txt